src/AltinnCore/Common/Languages/ini/nb.ini

```diff
@@ -300,6 +300,32 @@ wip_title = Her kommer det noe...
 all_forms = alle skjema
 inbox = innboks
 profile = profil
+party_selection_error_caption_prefix = Feil
```

Comment:
Consider putting these text keys in a "party_selection" section. This would shorten the keys a lot.

Reply:
Changed to what you suggested.